### PR TITLE
Update to 1.9.0

### PR DESCRIPTION
**Pull request by hfiguiere**
- also fix export of document icons

### CODE REVIEW

File: mapeditor-appdata.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- <id>org.mapeditor.Tiled.desktop</id>
++ <id>org.mapeditor.Tiled</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+ and BSD-2-Clause</project_license>
+  <name>Tiled</name>
+@@ -161,7 +161,6 @@
+  <url type="help">https://doc.mapeditor.org/</url>
+  <url type="donation">https://www.mapeditor.org/donate</url>
+  <url type="translate">https://hosted.weblate.org/projects/tiled/translations/</url>
+- <url type="vcs-browser">https://github.com/mapeditor/tiled</url>
```
Comment (bjorn) [inline mapeditor-appdata.patch]: Why remove this URL?

Comment (hfiguiere) [reply mapeditor-appdata.patch]: it cause the validation to fail. Not sure what changed with appstream again, but with this line it's fatal the build fails.

Comment (hfiguiere) [reply mapeditor-appdata.patch]: more like appstream-compose fail, which is much earlier.

**Comment by hfiguiere** [comment]
I removed the patch. It's a regression with the `flatpak-builder` command in Fedora SilverBlue that ours doesn't have.

**Comment by hfiguiere** [comment]
bot, build

File: mapeditor-appdata.patch (hunk 1 above)
Comment (hfiguiere) [reply mapeditor-appdata.patch]: I have removed this. It's caused by a discrepency between upstream flatpak-builder (that fails) and the version used on flathub. We need to get to the bottom of it, but not this time.

Comment (bjorn) [reply mapeditor-appdata.patch]: Ok, thanks!

**Comment by bjorn** [comment]
Thanks a lot for the update and the icon fixes!

**Pull request merged**